xstate/algorithm.py

```diff
@@ -458,6 +458,19 @@ def main_event_loop(machine: Machine, state: State, event: Event) -> State:
     return (configuration, actions)
 
 
+def execute_transition_content(
+    enabled_transitions: List[Transition], internal_queue: List[Event]
+):
+    actions: List[Action] = []
+    for transition in enabled_transitions:
+        for action in transition.actions:
+            actions.append(action)
+
+    for action in actions:
+        if action.type == "xstate:raise":
```

Comment:
In SCXML, when an event is **raised**, it is added to the `internal_queue`, which means that it's going to be "consumed" (sent to the machine) immediately before any other actions are taken or events are received.